src/api/client.ts

```
@@ -92,11 +153,17 @@ export class MotionApiClient {
     status?: string[];
     workspaceId?: string;
   }): Promise<MotionListResponse<MotionTask>> {
+    // Filter out includeAllStatuses if it's false
+    const filteredParams = params ? { ...params } : undefined;
+    if (filteredParams && filteredParams.includeAllStatuses === false) {
+      delete filteredParams.includeAllStatuses;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)